crates/turbo-tasks/src/read_ref.rs

```diff
@@ -130,17 +131,18 @@ impl<T, U: Serialize> Serialize for ReadRef<T, U> {
     }
 }
 
-impl<'de, T: Deserialize<'de>> Deserialize<'de> for ReadRef<T, T> {
+impl<'de, T, U: Deserialize<'de>> Deserialize<'de> for ReadRef<T, U> {
```

Comment:
Technically this is not sound since it allows you to construct any `ReadRef<T, U>` from deserialization, even where it's not guaranteed that T and U are binary identical. But I don't know a better way, so meh...